tools/applicationinsights-web-snippet/package.json

```diff
@@ -49,7 +50,6 @@
         "@nevware21/grunt-eslint-ts": "^0.2.2"
     },
     "dependencies": {
-        "@microsoft/dynamicproto-js": "^2.0.3",
-        "@microsoft/applicationinsights-common": "3.1.0"
+        "@microsoft/dynamicproto-js": "^2.0.3"
```

Comment:
It looks like this should move to a devDependency as well as this now has a dependency on ts-utils and I think this is only really used for rollup